openedx/adg/lms/applications/test/test_views.py

```diff
@@ -520,14 +538,17 @@ def test_post_with_no_cover_letter_typed_cover_letter_and_file_cover_letter_view
     a file, in each case the information is updated in the database
```

Comment:
Are we checking for the records in the database anywhere in the test function? If no, isn't the docstring a bit misleading?